capstone/src/main/java/com/google/univiz/collegescorecarddata/CollegeScoreCard.java

```diff
@@ -0,0 +1,19 @@
+package com.google.univiz.collegescorecarddata;
+
+import java.util.List;
+
+public interface CollegeScoreCard {
+  /** General Use Functions */
+  boolean validateCollegeName(String collegeName);
```

Comment:
What does this method do? How would an implementor know if a college name is valid?

Reply:
updated comments!